src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { Person } from '../types';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const getPeopleFromServer = async () => {
+    try {
+      setIsLoading(true);
+      const response = await getPeople();
+
+      setPeople(response);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    getPeopleFromServer();
+  }, []);
+
+  return (
+    <div className="block">
+      <h1 className="title">People Page</h1>
+
+      <div className="box table-container">
+        {isLoading && <Loader />}
+
+        {people && people.length > 0 && <PeopleTable people={people} />}
```

Comment:
you have to handle cases when you have 0 people in your table and when you get an error